static/docs/commands-reference/get.md

```diff
@@ -53,15 +53,16 @@ created in the current working directory, with its original file name.
 
 - `-v`, `--verbose` - displays detailed tracing information.
 
-## Examples
+## Example: Machine learning model deployment
```

Comment:
again, I think we should make titles for examples less specific ... at least for this one, we need to provide one very generic example

Reply:
> Also replying to comment from https://github.com/iterative/dvc.org/commit/8e9e94725675e5285d4d36f0a4fd8ae77c45e95b#r35084861:
>
> > let's use something more generic - Getting (or accessing? or downloading) model or data... Deployment scenario is too specific

OK. You had asked for that specific example before I think 😋 but agree, it's better as a side note, which I added in 5c3812e.